services/core/shared/core/delegates.js

```diff
@@ -206,14 +206,21 @@ const getNextForgers = async params => {
 const loadAllNextForgers = async () => {
 	// TODO: These feature should be handled by the compatibility layer
 	const maxCount = (sdkVersion < 4) ? 101 : 103;
-	rawNextForgers = await requestAll(coreApi.getNextForgers, { limit: maxCount }, maxCount);
-
+	if (sdkVersion <= 4) {
+		rawNextForgers = await requestAll(coreApi.getNextForgers, { limit: maxCount }, maxCount);
+	} else {
+		rawNextForgers = await coreApi.getForgers({ limit: maxCount, offset: nextForgers.length });
+	}
 	logger.info(`Updated next forgers list with ${rawNextForgers.length} delegates.`);
 };
 
 const resolveNextForgers = async () => {
-	nextForgers = await BluebirdPromise.map(rawNextForgers,
-		async forger => delegateList.find(o => o.address === forger.address));
+	if (sdkVersion <= 4) {
+		nextForgers = await BluebirdPromise.map(rawNextForgers,
+			async forger => delegateList.find(o => o.address === forger.address));
+	} else {
+		nextForgers = rawNextForgers.data;
+	}
```

Comment:
Please test if this works.

```suggestion
	nextForgers = await BluebirdPromise.map(
		rawNextForgers,
		async forger => sdkVersion <= 4
			? delegateList.find(o => o.address === forger.address)
			: forger,
	);
```

Reply:
Done!